decidim-verifications/app/controllers/decidim/verifications/authorizations_controller.rb

```diff
@@ -71,8 +71,17 @@ def handler_name
       def valid_handler
         return true if handler
 
-        logger.warn "Invalid authorization handler given: #{handler_name} doesn't"\
```

Comment:
I like the idea of raising in development, we probably should do that more 👍🏽 
But, can we leave the `logger.warn` also?
I'm worried about what would happen if anyone is adding a handler cowboy-style (like in preproduction or even production!) and they don't have any feedback.